### PR TITLE
fix #75685 economia.estadao.com.br

### LINKED ISSUE #75685: economia.estadao.com.br
### Issue URL (Anti Adblock Script)
 [https://economia.estadao.com.br/noticias/geral,presidente-da-petrobras-diz-que-custo-do-diesel-e-problema-da-idade-da-frota-de-caminhoes,70003597154](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Feconomia.estadao.com.br%2Fnoticias%2Fgeral%2Cpresidente-da-petrobras-diz-que-custo-do-diesel-e-problema-da-idade-da-frota-de-caminhoes%2C70003597154)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by stanislav-atr**
#75685

### CODE REVIEW

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -436,6 +436,7 @@ comnuan.com#%#//scriptlet("set-constant", "cmnnrunads", "true")
 marriedgames.com.br#%#//scriptlet("set-constant", "adning_no_adblock", "true")
 parasportontario.ca#%#//scriptlet("abort-on-property-read", "closeBlockerModal")
 ||studopedia.ru/ggl/$script
+estadao.com.br##.fc-ab-root
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/antiadblock.txt:439]: We have rules for it https://github.com/AdguardTeam/AdguardFilters/blob/10a8da3c87ef4b9239f9cfbfdb67baf73d9c84ac/AnnoyancesFilter/sections/antiadblock.txt#L391

Comment (Alex-302) [reply AnnoyancesFilter/sections/antiadblock.txt:439]: but I cannot reproduce both elements

Comment (zloyden) [reply AnnoyancesFilter/sections/antiadblock.txt:439]: Also not reproduced on my end.

Comment (stanislav-atr) [reply AnnoyancesFilter/sections/antiadblock.txt]: I lost the element, removed the rule. Cookie banner is still present ( espanol IPs):

<details><summary>Screenshot:</summary>

image
</details><br/>

**Review by zloyden — APPROVED** [review]

**Pull request merged**